### ISSUE TITLE
Zone.js testing dependencies missing?

### ISSUE THREAD

**Issue by kevinbeal** [wallabyjs/public#2689]
I keep getting a lot of missing dependency errors along the lines of the following:

> zone-testing.js is needed for the fakeAsync() test helper but could not be found. Please make sure that your environment includes zone.js/dist/zone-testing.js

I have an automatic configuration in an NX/Angular monorepo project.

This seemed to come out of nowhere. I just opened Webstorm in the morning. Maybe the plugin updated?

### Wallaby diagnostics report

```
{
  editorVersion: 'WebStorm 2020.3.3',
  pluginVersion: '1.0.216',
  editorType: 'IntelliJ',
  osVersion: 'linux 5.4.0-72-generic',
  nodeVersion: 'v12.3.1',
  coreVersion: '1.0.1062',
  checksum: 'ZjM2OTdjYjZhZTIwNzY2MzY4ZmM2NzM1M2NmNWI0ZWIsMTYzMDYyNzIwMDAwMCww',
  config: {
    env: {
      params: { runner: '--max_old_space_size=8192' },
      type: 'node',
      runner: '/usr/local/bin/node',
      viewportSize: { width: 800, height: 600 },
      options: { width: 800, height: 600 },
      bundle: true
    },
    diagnostics: {
      jest: {
        config: {
          configs: [
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'portal' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: {
                'ts-jest': {
                  tsConfig: '<rootDir>/tsconfig.spec.json',
                  stringifyContentPathRegex: '\\.(html|svg)$',
                  astTransformers: { before: [ 'jest-preset-angular/build/InlineFilesTransformer', 'jest-preset-angular/build/StripStylesTransformer' ] }
                }
              },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'js', 'html' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: '8ac533f090f2d1b5ff1bbe01af87b2a2',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/apps/portal',
              roots: [ '<homeDir>/www/underwriting-frontend1/apps/portal' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [ '<homeDir>/www/underwriting-frontend1/apps/portal/src/test-setup.ts' ],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [
                '<homeDir>/www/underwriting-frontend1/node_modules/jest-preset-angular/build/AngularNoNgAttributesSnapshotSerializer.js',
                '<homeDir>/www/underwriting-frontend1/node_modules/jest-preset-angular/build/AngularSnapshotSerializer.js',
                '<homeDir>/www/underwriting-frontend1/node_modules/jest-preset-angular/build/HTMLCommentSerializer.js'
              ],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [ [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ] ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'bootstrapper' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: {
                'ts-jest': {
                  tsConfig: '<rootDir>/tsconfig.spec.json',
                  stringifyContentPathRegex: '\\.(html|svg)$',
                  astTransformers: { before: [ 'jest-preset-angular/build/InlineFilesTransformer', 'jest-preset-angular/build/StripStylesTransformer' ] }
                }
              },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'js', 'html' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: 'be5422feb13deb78347e009e2f9f93e3',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/libs/bootstrapper',
              roots: [ '<homeDir>/www/underwriting-frontend1/libs/bootstrapper' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [ '<homeDir>/www/underwriting-frontend1/libs/bootstrapper/src/test-setup.ts' ],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [
                '<homeDir>/www/underwriting-frontend1/node_modules/jest-preset-angular/build/AngularNoNgAttributesSnapshotSerializer.js',
                '<homeDir>/www/underwriting-frontend1/node_modules/jest-preset-angular/build/AngularSnapshotSerializer.js',
                '<homeDir>/www/underwriting-frontend1/node_modules/jest-preset-angular/build/HTMLCommentSerializer.js'
              ],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [ [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ] ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'pdf-functions' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json' } },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'js', 'html' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: '7b3692383bf17a793c441bdcbedca33a',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/apps/pdf-functions',
              roots: [ '<homeDir>/www/underwriting-frontend1/apps/pdf-functions' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [ [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ] ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'utility' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json' } },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: 'ce9ed75bf5d798ac0542de077aeb2885',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/libs/utility',
              roots: [ '<homeDir>/www/underwriting-frontend1/libs/utility' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.[tj]sx?$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'app-designer' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: {},
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: 'f1cd57a5d262527ada49f64f28c9f301',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/apps/app-designer',
              roots: [ '<homeDir>/www/underwriting-frontend1/apps/app-designer' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)', '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/react/plugins/jest.js', {} ],
                [
                  '^.+\\.[tj]sx?$',
                  '<homeDir>/www/underwriting-frontend1/node_modules/jest-config/node_modules/babel-jest/build/index.js',
                  { cwd: '<homeDir>/www/underwriting-frontend1/apps/app-designer', configFile: './babel-jest.config.json' }
                ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'aframe' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json' } },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: '1dd98190288d6fccedd94915053d672d',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/libs/aframe',
              roots: [ '<homeDir>/www/underwriting-frontend1/libs/aframe' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.[tj]sx?$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'api-interfaces' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json' } },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: '3729fadcea1d421fb5cbbd535b8a72c1',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/libs/api-interfaces',
              roots: [ '<homeDir>/www/underwriting-frontend1/libs/api-interfaces' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.[tj]sx?$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'node-editor' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json' } },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: '575079c4478bd866f49555975078fba2',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/libs/node-editor',
              roots: [ '<homeDir>/www/underwriting-frontend1/libs/node-editor' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.[tj]sx?$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            },
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/www/underwriting-frontend1',
              dependencyExtractor: undefined,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: { color: 'white', name: 'components' },
              errorOnDeprecated: false,
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { 'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json' } },
              haste: { computeSha1: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx' ],
              moduleLoader: undefined,
              moduleNameMapper: [],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: '653a73f2bd040a3b9dbb7059afc89c8b',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: '<homeDir>/www/underwriting-frontend1/node_modules/@nrwl/jest/plugins/resolver.js',
              restoreMocks: false,
              rootDir: '<homeDir>/www/underwriting-frontend1/libs/components',
              roots: [ '<homeDir>/www/underwriting-frontend1/libs/components' ],
              runner: 'jest-runner',
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/www/underwriting-frontend1/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.[tj]sx?$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/www/underwriting-frontend1/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            }
          ],
          globalConfig: {
            bail: 0,
            changedFilesWithAncestor: false,
            changedSince: undefined,
            collectCoverage: false,
            collectCoverageFrom: [],
            collectCoverageOnlyFrom: undefined,
            coverageDirectory: '<homeDir>/www/underwriting-frontend1/coverage',
            coverageProvider: 'babel',
            coverageReporters: [ 'json', 'text', 'lcov', 'clover' ],
            coverageThreshold: undefined,
            detectLeaks: undefined,
            detectOpenHandles: undefined,
            enabledTestsMap: undefined,
            errorOnDeprecated: false,
            expand: false,
            filter: undefined,
            findRelatedTests: false,
            forceExit: false,
            globalSetup: undefined,
            globalTeardown: undefined,
            json: false,
            lastCommit: false,
            listTests: undefined,
            logHeapUsage: false,
            maxConcurrency: 5,
            maxWorkers: 7,
            noSCM: undefined,
            noStackTrace: false,
            nonFlagArgs: undefined,
            notify: false,
            notifyMode: 'failure-change',
            onlyChanged: false,
            onlyFailures: false,
            outputFile: undefined,
            passWithNoTests: undefined,
            projects: [
              '<homeDir>/www/underwriting-frontend1/apps/portal',
              '<homeDir>/www/underwriting-frontend1/libs/bootstrapper',
              '<homeDir>/www/underwriting-frontend1/apps/pdf-functions',
              '<homeDir>/www/underwriting-frontend1/libs/utility',
              '<homeDir>/www/underwriting-frontend1/apps/app-designer',
              '<homeDir>/www/underwriting-frontend1/libs/aframe',
              '<homeDir>/www/underwriting-frontend1/libs/api-interfaces',
              '<homeDir>/www/underwriting-frontend1/libs/node-editor',
              '<homeDir>/www/underwriting-frontend1/libs/components'
            ],
            replname: undefined,
            reporters: undefined,
            rootDir: '<homeDir>/www/underwriting-frontend1',
            runTestsByPath: false,
            silent: undefined,
            skipFilter: false,
            testFailureExitCode: 1,
            testNamePattern: undefined,
            testPathPattern: '',
            testResultsProcessor: undefined,
            testSequencer: '<homeDir>/www/underwriting-frontend1/node_modules/@jest/test-sequencer/build/index.js',
            testTimeout: undefined,
            updateSnapshot: 'new',
            useStderr: false,
            verbose: undefined,
            watch: false,
            watchAll: false,
            watchPlugins: undefined,
            watchman: true
          },
          hasDeprecationWarnings: false,
          wallaby: {
            roots: [ 'apps/portal', 'libs/bootstrapper', 'apps/pdf-functions', 'libs/utility', 'apps/app-designer', 'libs/aframe', 'libs/api-interfaces', 'libs/node-editor', 'libs/components' ],
            watchPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|/\\..+/' ],
            testPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|/\\..+/' ],
            testMatch: [ '**/+(*.)+(spec).+(ts|js)?(x)' ],
            testRegex: []
          }
        }
      }
    },
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
    filesWithCoverageCalculated: [],
    filesWithNoCoverageCalculated: [],
    globalSetup: false,
    micromatch: true,
    files: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true },
      { pattern: '\\./dist/|\\./build/|\\./coverage/|/\\..+/', regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\/\..+\//, ignore: true, trigger: true, load: true },
      { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: 'apps/portal/**', ignore: false, trigger: true, load: true, order: 2 },
      { pattern: 'libs/bootstrapper/**', ignore: false, trigger: true, load: true, order: 3 },
      { pattern: 'apps/pdf-functions/**', ignore: false, trigger: true, load: true, order: 4 },
      { pattern: 'libs/utility/**', ignore: false, trigger: true, load: true, order: 5 },
      { pattern: 'apps/app-designer/**', ignore: false, trigger: true, load: true, order: 6 },
      { pattern: 'libs/aframe/**', ignore: false, trigger: true, load: true, order: 7 },
      { pattern: 'libs/api-interfaces/**', ignore: false, trigger: true, load: true, order: 8 },
      { pattern: 'libs/node-editor/**', ignore: false, trigger: true, load: true, order: 9 },
      { pattern: 'libs/components/**', ignore: false, trigger: true, load: true, order: 10 },
      { pattern: 'apps/portal/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'libs/bootstrapper/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'apps/pdf-functions/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'libs/utility/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'apps/app-designer/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'libs/aframe/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'libs/api-interfaces/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'libs/node-editor/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'libs/components/**/+(*.)+(spec).+(ts|js)?(x)', ignore: true, trigger: true, load: true }
    ],
    tests: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, test: true },
      { pattern: '\\./dist/|\\./build/|\\./coverage/|/\\..+/', regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\/\..+\//, ignore: true, trigger: true, load: true, test: true },
      { pattern: 'apps/portal/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 11 },
      { pattern: 'libs/bootstrapper/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 12 },
      { pattern: 'apps/pdf-functions/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 13 },
      { pattern: 'libs/utility/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 14 },
      { pattern: 'apps/app-designer/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 15 },
      { pattern: 'libs/aframe/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 16 },
      { pattern: 'libs/api-interfaces/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 17 },
      { pattern: 'libs/node-editor/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 18 },
      { pattern: 'libs/components/**/+(*.)+(spec).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 19 }
    ],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    addModifiedTestFileToExclusiveTestRun: true,
    compilers: {},
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'auto.detect#-1745333141'
  },
  packageJSON: {
    dependencies: {
      '@angular/animations': '11.2.1',
      '@angular/bazel': '^11.0.4',
      '@angular/cdk': '^11.1.1',
      '@angular/common': '11.2.1',
      '@angular/compiler': '11.2.1',
      '@angular/core': '11.2.1',
      '@angular/forms': '11.2.1',
      '@angular/platform-browser': '11.2.1',
      '@angular/platform-browser-dynamic': '11.2.1',
      '@angular/router': '11.2.1',
      '@bugsnag/js': '^7.2.1',
      '@bugsnag/plugin-angular': '^7.2.1',
      '@mdi/font': '^5.5.55',
      '@nestjs/common': '^7.0.0',
      '@nestjs/core': '^7.0.0',
      '@nestjs/platform-express': '^7.0.0',
      '@ngrx/component-store': '11.0.0',
      '@ngrx/effects': '11.0.0',
      '@ngrx/entity': '11.0.0',
      '@ngrx/store': '11.0.0',
      '@nrwl/angular': '11.3.1',
      'auth0-js': '^9.13.4',
      'aws-lambda': '^1.0.6',
      'aws-serverless-express': '^3.4.0',
      'body-parser': '^1.19.0',
      'classlist.js': '^1.1.20150312',
      'core-js': '^3.6.5',
      cors: '^2.8.5',
      'css-vars-ponyfill': '^2.3.2',
      'document-register-element': '1.13.1',
      helmet: '^4.4.1',
      'html-to-image': '^0.1.1',
      immer: '^8.0.1',
      luxon: '^1.24.1',
      'mutationobserver-shim': '^0.3.7',
      'ngx-color-picker': '^9.1.0',
      'ngx-markdown': '^10.1.0',
      'normalize.css': '^8.0.1',
      'pdf-lib': '^1.14.1',
      'pdfjs-dist': '2.6.347',
      pdfkit: '^0.10.0',
      react: '17.0.1',
      'react-dom': '17.0.1',
      'react-is': '17.0.1',
      'react-router-dom': '5.2.0',
      'reflect-metadata': '^0.1.13',
      rxjs: '^6.6.0',
      'rxjs-compat': '^6.6.0',
      'styled-components': '5.2.1',
      tslib: '^2.0.0',
      'web-animations-js': '^2.3.2',
      'zone.js': '~0.10.2'
    },
    devDependencies: {
      '@angular-devkit/build-angular': '0.1102.1',
      '@angular-devkit/core': '^11.1.2',
      '@angular-devkit/schematics': '^11.1.2',
      '@angular/cli': '11.2.1',
      '@angular/compiler-cli': '11.2.1',
      '@angular/language-service': '11.2.1',
      '@babel/core': '7.9.6',
      '@babel/plugin-proposal-class-properties': '^7.12.13',
      '@babel/plugin-proposal-nullish-coalescing-operator': '^7.12.13',
      '@babel/plugin-proposal-optional-chaining': '^7.12.17',
      '@babel/preset-env': '7.9.6',
      '@babel/preset-react': '7.9.4',
      '@babel/preset-typescript': '^7.9.0',
      '@cypress/webpack-preprocessor': '^5.4.2',
      '@nestjs/schematics': '^7.0.0',
      '@nestjs/testing': '^7.0.0',
      '@ngrx/schematics': '11.0.0',
      '@ngrx/store-devtools': '11.0.0',
      '@nrwl/cypress': '11.3.1',
      '@nrwl/eslint-plugin-nx': '11.3.1',
      '@nrwl/jest': '11.3.1',
      '@nrwl/linter': '11.3.1',
      '@nrwl/nest': '11.3.1',
      '@nrwl/node': '11.3.1',
      '@nrwl/nx-cloud': '11.1.4',
      '@nrwl/react': '11.3.1',
      '@nrwl/web': '11.3.1',
      '@nrwl/workspace': '11.3.1',
      '@testing-library/react': '11.1.2',
      '@types/auth0-js': '^9.13.4',
      '@types/aws-lambda': '^8.10.71',
      '@types/aws-serverless-express': '^3.3.3',
      '@types/chart.js': '^2.9.23',
      '@types/cors': '^2.8.9',
      '@types/jest': '26.0.8',
      '@types/luxon': '^1.24.3',
      '@types/node': '12.12.38',
      '@types/node-forge': '^0.9.7',
      '@types/pdfkit': '^0.10.6',
      '@types/react': '16.9.56',
      '@types/react-dom': '16.9.9',
      '@types/react-is': '16.7.1',
      '@types/react-router-dom': '5.1.6',
      '@types/styled-components': '5.1.4',
      '@typescript-eslint/eslint-plugin': '4.3.0',
      '@typescript-eslint/parser': '4.3.0',
      'angular2-template-loader': '^0.6.2',
      'babel-jest': '26.2.2',
      'babel-plugin-styled-components': '1.10.7',
      codelyzer: '6.0.1',
      cypress: '^5.5.0',
      dotenv: '^8.2.0',
      eslint: '7.10.0',
      'eslint-config-prettier': '6.0.0',
      'eslint-plugin-cypress': '^2.10.3',
      'eslint-plugin-import': '2.22.1',
      'eslint-plugin-jest': '^24.3.5',
      'eslint-plugin-jsx-a11y': '6.4.1',
      'eslint-plugin-react': '7.21.5',
      'eslint-plugin-react-hooks': '4.2.0',
      jest: '26.2.2',
      'jest-circus': '^26.4.2',
      'jest-marbles': '^2.5.1',
      'jest-preset-angular': '8.3.2',
      'ng-packagr': '11.2.2',
      prettier: '2.2.1',
      'rxjs-watcher': '^1.1.3',
      'shallow-render': '^10.2.1-0',
      stylelint: '^13.6.1',
      'ts-jest': '26.4.0',
      'ts-loader': '^8.0.1',
      'ts-node': '9.1.1',
      tslint: '6.1.3',
      'tslint-microsoft-contrib': '^6.2.0',
      'tslint-sonarts': '^1.9.0',
      typescript: '^4.0.5',
      webpack: '^4.44.1'
    }
  },
  fs: { numberOfFiles: 2269 },
  debug: [
    '2021-04-19T20:34:52.402Z angular/cli config Detected Angular CLI.\n',
    '2021-04-19T20:34:52.404Z angular/cli config @nrwl/workspace v11.1.4 and higher uses jest automatic configuration.\n',
    '2021-04-19T20:34:52.830Z jest/config Detected Jest.\n',
    '2021-04-19T20:34:52.831Z jest/config Configured Jest.\n',
    '2021-04-19T20:34:52.833Z project Wallaby Node version: v12.3.1\n',
    '2021-04-19T20:34:52.833Z project Wallaby config: <homeDir>/www/underwriting-frontend1/auto.detect\n',
    '2021-04-19T20:34:54.788Z project File cache: <homeDir>/.cache/JetBrains/WebStorm2020.3/wallaby/projects/5074213c43fd5523\n',
    '2021-04-19T20:34:54.846Z uiService Listening port 51235\n',
    '2021-04-19T20:34:54.924Z project Config file change detected, invalidating local cache\n',
    '2021-04-19T20:34:55.415Z workers Parallelism for initial run: 6, for regular run: 3\n',
    '2021-04-19T20:34:55.415Z workers Starting run worker instance #0\n',
    '2021-04-19T20:34:55.415Z workers Starting run worker instance #1\n',
    '2021-04-19T20:34:55.415Z workers Starting run worker instance #2\n',
    '2021-04-19T20:34:55.415Z workers Starting run worker instance #3\n',
    '2021-04-19T20:34:55.415Z workers Starting run worker instance #4\n',
    '2021-04-19T20:34:55.415Z workers Starting run worker instance #5\n',
    '2021-04-19T20:34:55.416Z workers Web server is listening at 35739\n',
    '2021-04-19T20:34:55.416Z project File cache requires some updates, waiting required files from IDE\n',
    '2021-04-19T20:34:55.838Z workers Started run worker instance (delayed) #1\n',
    '2021-04-19T20:34:55.839Z workers Started run worker instance (delayed) #0\n',
    '2021-04-19T20:34:55.839Z workers Started run worker instance (delayed) #2\n',
    '2021-04-19T20:34:55.839Z workers Started run worker instance (delayed) #3\n',
    '2021-04-19T20:34:55.846Z workers Started run worker instance (delayed) #4\n',
    '2021-04-19T20:34:55.846Z workers Started run worker instance (delayed) #5\n',
    '2021-04-19T20:34:56.023Z project Stopping process pool\n',
    '2021-04-19T20:34:56.028Z project Test run started; run priority: 3\n',
    '2021-04-19T20:34:56.049Z project Running all tests\n',
    '2021-04-19T20:34:56.093Z workers Starting test run, priority: 3\n',
    '2021-04-19T20:34:56.093Z workers Distributing tests between 6 workers\n',
    '2021-04-19T20:34:56.099Z workers Running tests in parallel\n',
    '2021-04-19T20:34:56.100Z nodeRunner Starting sandbox [worker #0, session #70aog]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Starting sandbox [worker #1, session #beugz]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Starting sandbox [worker #2, session #3wf4c]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Starting sandbox [worker #3, session #k9jk5]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Starting sandbox [worker #4, session #rh818]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Starting sandbox [worker #5, session #eb84d]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Preparing sandbox [worker #0, session #70aog]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Preparing sandbox [worker #1, session #beugz]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Preparing sandbox [worker #2, session #3wf4c]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Preparing sandbox [worker #3, session #k9jk5]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Preparing sandbox [worker #4, session #rh818]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Preparing sandbox [worker #5, session #eb84d]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Prepared sandbox [worker #0, session #70aog]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Prepared sandbox [worker #1, session #beugz]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Prepared sandbox [worker #2, session #3wf4c]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Prepared sandbox [worker #3, session #k9jk5]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Prepared sandbox [worker #4, session #rh818]\n',
    '2021-04-19T20:34:56.100Z nodeRunner Prepared sandbox [worker #5, session #eb84d]\n',
    '2021-04-19T20:34:56.101Z workers [worker #0, session #70aog] Running tests in sandbox\n',
    '2021-04-19T20:34:56.105Z workers [worker #1, session #beugz] Running tests in sandbox\n',
    '2021-04-19T20:34:56.113Z workers [worker #2, session #3wf4c] Running tests in sandbox\n',
    '2021-04-19T20:34:56.117Z workers [worker #3, session #k9jk5] Running tests in sandbox\n',
    '2021-04-19T20:34:56.121Z workers [worker #4, session #rh818] Running tests in sandbox\n',
    '2021-04-19T20:34:56.124Z workers [worker #5, session #eb84d] Running tests in sandbox\n',
    '2021-04-19T20:35:42.246Z workers Scheduling Jest Test Run: 2021-04-19T20:34:57.485Z\n',
    '2021-04-19T20:35:42.416Z workers Scheduling Jest Test Run: 2021-04-19T20:34:57.505Z\n',
    '2021-04-19T20:35:42.757Z workers Scheduling Jest Test Run: 2021-04-19T20:34:57.247Z\n',
    '2021-04-19T20:35:43.201Z workers Scheduling Jest Test Run: 2021-04-19T20:34:57.397Z\n',
    '2021-04-19T20:35:43.316Z workers Scheduling Jest Test Run: 2021-04-19T20:34:57.325Z\n',
    '2021-04-19T20:35:48.748Z workers Scheduling Jest Test Run: 2021-04-19T20:34:57.321Z\n',
    '2021-04-19T20:37:22.910Z workers [70aog] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:22.919Z workers [70aog] Test executed: should be created\n',
    '2021-04-19T20:37:22.921Z workers Failed to map the stack to user code, entry message: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js, stack: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js\n    at resetFakeAsyncZone (<homeDir>/www/packages/core/testing/src/fake_async.ts:25:9)\n    at ' +
      'Object.<anonymous> (<homeDir>/www/packages/core/testing/src/before_each.ts:26:5)\n    at Object.asyncJestLifecycle ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:58:37)\n    at ' +
      '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:45:12\n    at new Promise (<anonymous>)\n    at mapper ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:28:19)\n    at ' +
      '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:75:41\n',
    '2021-04-19T20:37:23.932Z workers [beugz] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:23.933Z workers [beugz] Test executed: should have instantiated\n',
    '2021-04-19T20:37:23.934Z workers Failed to map the stack to user code, entry message: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js, stack: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js\n    at resetFakeAsyncZone (<homeDir>/www/packages/core/testing/src/fake_async.ts:25:9)\n    at ' +
      'Object.<anonymous> (<homeDir>/www/packages/core/testing/src/before_each.ts:26:5)\n    at Object.asyncJestLifecycle ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:58:37)\n    at ' +
      '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:45:12\n    at new Promise (<anonymous>)\n    at mapper ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:28:19)\n    at ' +
      '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:75:41\n',
    '2021-04-19T20:37:24.612Z workers [beugz] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.615Z workers [beugz] Test executed: should add box if show truthy or hide falsy\n',
    '2021-04-19T20:37:24.618Z workers [beugz] Test executed: should error if no logic found\n',
    '2021-04-19T20:37:24.619Z workers [beugz] Test executed: should error if logic errored\n',
    '2021-04-19T20:37:24.639Z workers [beugz] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.644Z workers [beugz] Test executed: Update the value\n',
    '2021-04-19T20:37:24.644Z workers [k9jk5] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.644Z workers [beugz] Test executed: should add an emptied object in step-repeated list after index\n',
    '2021-04-19T20:37:24.644Z workers [k9jk5] Test executed: should be created\n',
    '2021-04-19T20:37:24.645Z workers Failed to map the stack to user code, entry message: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js, stack: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js\n    at resetFakeAsyncZone (<homeDir>/www/packages/core/testing/src/fake_async.ts:25:9)\n    at ' +
      'Object.<anonymous> (<homeDir>/www/packages/core/testing/src/before_each.ts:26:5)\n    at Object.asyncJestLifecycle ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:58:37)\n    at ' +
      '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:45:12\n    at new Promise (<anonymous>)\n    at mapper ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:28:19)\n    at ' +
      '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:75:41\n',
    '2021-04-19T20:37:24.646Z workers [beugz] Test executed: should add an emptied object in step-repeated list after index\n',
    '2021-04-19T20:37:24.647Z workers [beugz] Test executed: should add parsed state\n',
    '2021-04-19T20:37:24.648Z workers [beugz] Test executed: should change the active step\n',
    '2021-04-19T20:37:24.650Z workers [beugz] Test executed: should update the fields\n',
    '2021-04-19T20:37:24.650Z workers [beugz] Test executed: should return the initial state\n',
    '2021-04-19T20:37:24.679Z workers [k9jk5] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.695Z workers [k9jk5] Test executed: should take logic config with truthy outputValue in output.steps\n',
    '2021-04-19T20:37:24.701Z workers [k9jk5] Test executed: should take logic config with no or falsy in case of hide\n',
    '2021-04-19T20:37:24.702Z workers [k9jk5] Test executed: should disable-able\n',
    '2021-04-19T20:37:24.707Z workers [k9jk5] Test executed: should error if no condition was found for hide logic id\n',
    '2021-04-19T20:37:24.710Z workers [k9jk5] Test executed: should error if logic errored\n',
    '2021-04-19T20:37:24.710Z workers [k9jk5] Test executed: should update the fields if they were mutated\n',
    '2021-04-19T20:37:24.713Z workers [k9jk5] Test executed: should error if no steps are visible\n',
    '2021-04-19T20:37:24.720Z workers [k9jk5] Test executed: should update the active step if no longer visible\n',
    '2021-04-19T20:37:24.737Z workers [k9jk5] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.737Z workers [k9jk5] Test executed: should generate a summary\n',
    '2021-04-19T20:37:24.744Z workers [k9jk5] Test executed: should build a summary\n',
    '2021-04-19T20:37:24.748Z workers [k9jk5] Test executed: should ignore text blocks\n',
    '2021-04-19T20:37:24.749Z workers [k9jk5] Test executed: should build a summary\n',
    '2021-04-19T20:37:24.750Z workers [k9jk5] Test executed: should get progress from its inputs\n',
    '2021-04-19T20:37:24.751Z workers [k9jk5] Test executed: should build a summary\n',
    '2021-04-19T20:37:24.751Z workers [k9jk5] Test executed: should build a summary\n',
    '2021-04-19T20:37:24.754Z workers [k9jk5] Test executed: should handle number fields\n',
    '2021-04-19T20:37:24.757Z workers [k9jk5] Test executed: should handle dropdown fields\n',
    '2021-04-19T20:37:24.759Z workers [k9jk5] Test executed: should handle radio fields\n',
    '2021-04-19T20:37:24.760Z workers [k9jk5] Test executed: should handle phone fields\n',
    '2021-04-19T20:37:24.769Z workers [k9jk5] Test executed: should handle date fields\n',
    '2021-04-19T20:37:24.775Z workers [70aog] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.776Z workers [k9jk5] Test executed: should handle checkbox fields\n',
    '2021-04-19T20:37:24.776Z workers [70aog] Test executed: should update the output.blocks if logic true\n',
    '2021-04-19T20:37:24.780Z workers [k9jk5] Test executed: should handle text fields\n',
    '2021-04-19T20:37:24.782Z workers [70aog] Test executed: should update the output.blocks if no logic of logic false for hide\n',
    '2021-04-19T20:37:24.785Z workers [k9jk5] Test executed: should handle empty values\n',
    '2021-04-19T20:37:24.788Z workers [70aog] Test executed: should add block to disabled list if condition true\n',
    '2021-04-19T20:37:24.788Z workers [k9jk5] Test executed: should handle string values\n',
    '2021-04-19T20:37:24.788Z workers [70aog] Test executed: should disable all blocks if step is disabled\n',
    '2021-04-19T20:37:24.789Z workers [k9jk5] Test executed: should handle case of zero objects\n',
    '2021-04-19T20:37:24.789Z workers [70aog] Test executed: should complain if logic with id is missing\n',
    "2021-04-19T20:37:24.791Z workers [k9jk5] Test executed: should just give it if there's just one\n",
    '2021-04-19T20:37:24.791Z workers [70aog] Test executed: should error if parsing errors\n',
    '2021-04-19T20:37:24.793Z workers [k9jk5] Test executed: should combine multiple\n',
    '2021-04-19T20:37:24.806Z workers [k9jk5] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.807Z workers [k9jk5] Test executed: create an instance\n',
    '2021-04-19T20:37:24.813Z workers [k9jk5] Test executed: should find DateTime dates in a list of them\n',
    '2021-04-19T20:37:24.854Z workers [70aog] Loaded unknown number of test(s)\n',
    '2021-04-19T20:37:24.857Z workers [70aog] Test executed: should error if event type is unsupported\n',
    '2021-04-19T20:37:24.857Z workers Failed to map the stack to user code, entry message: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js, stack: Error: zone-testing.js is needed for the fakeAsync() test helper but could not be found.\n        ' +
      'Please make sure that your environment includes zone.js/dist/zone-testing.js\n    at resetFakeAsyncZone (<homeDir>/www/packages/core/testing/src/fake_async.ts:25:9)\n    at ' +
      'Object.<anonymous> (<homeDir>/www/packages/core/testing/src/before_each.ts:26:5)\n    at Object.asyncJestLifecycle ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:58:37)\n    at ' +
      '<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:45:12\n    at new Promise (<anonymous>)\n    at mapper ' +
      '(<homeDir>/www/underwriting-frontend1/node_modules/jest-jasmine2/build/queueRunner.js:28:19)\n    at ' +
...
```

**Comment by smcenlly** [comment]
Could you please confirm for us that you are able to run your tests from the command-line of your mono repo using `npx jest`?

**Comment by kevinbeal** [comment]
Yes. It runs tests without error. It only runs 28 of 2600+ though.

**Comment by smcenlly** [comment]
We recently made some improvements to our jest project loading. It's unlikely (but possible) that this is causing a problem for you. Could you please follow our [offline installation instructions](https://wallabyjs.com/docs/intro/offline.html) and install version 1.0.61 for Wallaby core (Skip the step that asks you to get the latestServer version).

In the meantime, we're setting up an nx workspace with similar dependency versions to yours to see if we can replicate the problem ourselves.

**Comment by smcenlly** [comment]
We have reproduced the problem. Your problem is almost certainly the same and introduced by a recent jest improvement that we made. Please update to Wallaby core `v1.0.1064` to fix your issue.

**Comment by smcenlly** [comment]
Please note that the offline instruction request was superseded by my latest comment:

> We have reproduced the problem. Your problem is almost certainly the same and introduced by a recent jest improvement that we made. Please update to Wallaby core v1.0.1064 to fix your issue.

Please undo any steps you performed to attempt to install Wallaby offline. The latest version of the Wallaby extension and Wallaby core (v1.0.1064+) should fix your problem.

**Comment by kevinbeal** [comment]
Oh. Right. Will do. Thanks for the quick work!
